debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresType.java

```diff
@@ -94,6 +99,44 @@ public int getDefaultScale() {
         return defaultScale;
     }
 
+    /**
+     * @param modifier - type modifier coming from decoder
+     * @return length of the type based on the modifier
+     */
+    public int length(int modifier) {
```

Comment:
So in the case where we don't receive any length metadata in the message for a `NUMERIC(x,y)` column, won't this cause to fallback to the default of the oid's values instead of keeping the values from that specific column (as read via JDBC initially)?

Reply:
This method is calle dand used if and only if the type modifiers metadat are included in the message.

Reply:
Ok, I can confirm that. There was a super-confusing usage of `getTypeMetadata()` which was passed to `getValue()` but then never used. I've pushed another commit for getting rid of this.

Reply:
Fine for me, probably leftover after refactoring